uvsim.py

```diff
@@ -53,3 +53,19 @@ def execute(self):
         except KeyboardInterrupt:
             print("\nAborting...")
             exit(0)
+
+    def store(self, filename):
+        """
+        Store the contents of memory to a file
+        """
+        with open(filename, 'w') as program:
```

Comment:
Thanks for working on this. I think it would be preferable to iterate through memory directly though. Also the memory object is always technically 100  in length. `len(self.mem)` will currently only read the number of writes to memory that are not zero. 

Try something like:

```python
for opcode in self.mem:
   program.write(str(opcode))
```

But I haven't actually tested that. It's just a little odd to have to implement the counter by hand. If it's not trivial to write the for loop that way, leave what you have here but triple check that it writes the file the way you'd expect. 

Additional test cases to consider:
- if a program is has the opcode `+0000` in between regular opcodes like:
```
+1010
+0000
+0000
+2310
+0000
+3210
```

Do they get written to the correct line of the file?